models/place.rb

```diff
@@ -0,0 +1,15 @@
+require "sequel"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.